apps/frontend/src/app/pages/admin/admin-activity/admin-activity.component.ts

```diff
@@ -0,0 +1,118 @@
+import { Component, OnInit } from '@angular/core';
+import { SharedModule } from '../../../shared.module';
+import { SpinnerDirective } from '../../../directives';
+import { AdminActivityService } from '../../../services';
+import { AdminActivity, AdminActivityType } from '@momentum/constants';
+import { Subject, merge, of, switchMap, tap } from 'rxjs';
+import { MessageService } from 'primeng/api';
+import { PaginatorModule, PaginatorState } from 'primeng/paginator';
+import {
+  AdminActivityEntryComponent,
+  AdminActivityEntryData
+} from './admin-activity-entry/admin-activity-entry.component';
+import { AdminActivityEntryHeaderComponent } from './admin-activity-entry/admin-activity-entry-header.component';
+import {
+  AccordionComponent,
+  AccordionItemComponent
+} from '../../../components';
+import { ActivatedRoute } from '@angular/router';
+
+@Component({
+  selector: 'm-admin-activity',
+  templateUrl: './admin-activity.component.html',
+  standalone: true,
+  imports: [
+    SharedModule,
+    SpinnerDirective,
+    PaginatorModule,
+    AdminActivityEntryComponent,
+    AdminActivityEntryHeaderComponent,
+    AccordionComponent,
+    AccordionItemComponent
+  ]
+})
+export class AdminActivityComponent implements OnInit {
+  constructor(
+    private readonly route: ActivatedRoute,
+    private readonly adminActivityService: AdminActivityService,
+    private readonly toasterService: MessageService
+  ) {}
+
+  readonly ADMIN_ACTIVITIES_FILTERS = [
+    { value: undefined, text: 'All' },
+    { value: AdminActivityType.USER_UPDATE_ROLES, text: 'Roles update' },
+    { value: AdminActivityType.USER_UPDATE_BANS, text: 'Bans update' },
+    { value: AdminActivityType.USER_UPDATE_ALIAS, text: 'Alias update' },
+    { value: AdminActivityType.USER_UPDATE_BIO, text: 'Bio update' },
+    {
+      value: AdminActivityType.USER_CREATE_PLACEHOLDER,
+      text: 'Placeholder created'
+    },
+    { value: AdminActivityType.USER_MERGE, text: 'User merged' },
+    { value: AdminActivityType.USER_DELETE, text: 'User deleted' },
+    { value: AdminActivityType.MAP_UPDATE, text: 'Map update' },
+    { value: AdminActivityType.MAP_DELETE, text: 'Map deleted' },
+    { value: AdminActivityType.REPORT_UPDATE, text: 'Report update' },
+    { value: AdminActivityType.REPORT_RESOLVE, text: 'Report resolve' }
+  ];
+
+  activities: { activity: AdminActivity; entry: AdminActivityEntryData }[] = [];
+
+  loading: boolean;
+  readonly pageChange = new Subject<PaginatorState>();
+  readonly filterChange = new Subject<void>();
+
+  readonly rows = 10;
+  totalRecords = 0;
+  first = 0;
+  filter = undefined;
```

Comment:
Use `protected` for something the component can see, `private` if not, explicit `public` if it should be visible to parent components. I know I'm nitpicking, but I really like making it clear what's exposes to the component and what isn't.